05_customizing-extending-masa/06_events/01_event-hooks.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: default
+title: Event hooks
+nav_order: 1
+grand_parent: Customizing/Extending Masa
+parent: Events
+last_modified_date: 11-01-2021 10:00
+permalink: /customizing-extending-masa/events/event-hooks/
+---
+
+# Event hooks
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+- TOC
+{:toc}
+
+## Standard events
```

Comment:
Soo you've listed all the event, which is great. Perhas a short description is needed to explain what Event Hooks are and how they can be used?